packages/hierarchies-react/src/presentation-hierarchies-react/UseTree.ts

```diff
@@ -23,6 +23,10 @@ import { SelectionChangeType } from "./UseSelectionHandler";
 /** @beta */
 export interface HierarchyLevelConfiguration {
   hierarchyNode: HierarchyNode | undefined;
+  getInstanceKeysIterator: (props?: {
+    instanceFilter?: GenericInstanceFilter;
+    hierarchyLevelSizeLimit?: number | "unbounded";
+  }) => AsyncIterableIterator<InstanceKey>;
```

Comment:
Somehow this function doesn't fit into what I understand as "configuration". Could it be moved to `UseTreeResult` maybe? Or `HierarchyLevelConfiguration` needs some other name.

Reply:
Renamed to `HierarchyLevelDetails`. Couldn't come up with better name. 😆 I would like to keep it on `HierarchyLevelConfiguration/Details`. Putting it directly on `UseTreeResult` doesn't allow us to hide parent hierarchy node and requires consumers to pass `HierarchyNode` for which instance keys should be loaded.